EasyVCR.Tests/ClientTest.cs

```diff
@@ -179,6 +180,43 @@ public async Task TestEraseAndRecord()
             Assert.IsTrue(cassette.NumInteractions > 0); // Make sure cassette is not empty
         }
 
+        [TestMethod]
+        public async Task TestExpirationSettings()
+        {
+            var cassette = TestUtils.GetCassette("test_expiration_settings");
+            cassette.Erase(); // Erase cassette before recording
+
+            // record cassette first
+            var client = HttpClients.NewHttpClient(cassette, Mode.Record);
+            var fakeDataService = new FakeJsonDataService(client);
+            var _ = await fakeDataService.GetExchangeRatesRawResponse();
```

Comment:
```suggestion
            await fakeDataService.GetExchangeRatesRawResponse();
```

Remove the variable assignment if it's not going to be used